### ISSUE TITLE
winget installation failed

### ISSUE THREAD

**Issue by jrnewton** [AmrDeveloper/GQL#48]
**Describe the bug**
Installation using ```winget install gitql``` produces an error:
```
Installer failed with exit code: 0x80070002 : The system cannot find the file specified.
```

**To Reproduce**
1. Open cmd window with admin permissions
2. Run ```winget install gitql```

**Expected behavior**
I expected the package to install.  Instead an error was produced.

Full output
```
> winget install gitql
Found GitQL [amrdeveloper.gitql] Version 0.7.1
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
This package requires the following dependencies:
  - Packages
      Microsoft.VCRedist.2015+.x64
      Microsoft.UI.Xaml.2.8
Installing dependencies:
(1/1) Found Microsoft.UI.Xaml [Microsoft.UI.Xaml.2.8] Version 8.2306.22001.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Successfully verified installer hash
Starting package install...
  ████████████████████████████▌   95%
Installer failed with exit code: 0x80070002 : The system cannot find the file specified.
```
 - Version [e.g. 0.7.1]

**Comment by allout58** [comment]
I get something similar: `Installer failed with exit code: 0x80073d06 : The package could not be installed because a higher version of this package is already installed.`.

I've had issues with Microsoft.UI.Xaml.2.8 for other installs with WinGet (like https://github.com/microsoft/winget-pkgs/issues/127953). You can try to bypass it with `--skip-dependencies` to see if that works.

**Comment by samchon** [comment]
In my case, such unreadable message comes:

<img width="867" alt="image" src="https://github.com/AmrDeveloper/GQL/assets/13158709/a97416f0-6b01-4662-9b91-e027ae180120">

**Comment by AmrDeveloper** [comment]
Winget should be working without any issues now for version `0.10.`

https://github.com/microsoft/winget-pkgs/pull/129708#issuecomment-1867862419

**Comment by AmrDeveloper** [comment]
Fixed on `0.10.0` and above
